cmd/atlas/internal/migrate/ent/schema/revision.go

```diff
@@ -15,6 +15,9 @@ import (
 	"entgo.io/ent/schema/field"
 )
 
+// DefaultRevisionSchema is the default schema for storing revisions table.
+const DefaultRevisionSchema = "atlas_schema_revisions"
```

Comment:
If it's a schema, why do you use it as a table in the annotation below?

Reply:
Both are named the same.